CONTRIBUTING.md

```diff
@@ -432,8 +432,8 @@ release, either update to Windows 10/11 or move to macOS/Linux/BSD.
 In order to speed up patching process on Windows, it's recommended you get WSL
 2. This is available in Windows 10 v2004, build 19041 or higher. (You can check
 your version by running `winver` in the run window (Windows key + R)). If you're
-in Windows 10 and out of date, update your system with the
-[Windows 10 Update Assistant](https://www.microsoft.com/en-us/software-download/windows10).
+using an out of date version of Windows 10, update your system with the
+[Windows 10 Update Assistant](https://www.microsoft.com/en-us/software-download/windows10) or upgrade with [Windows 11 Update Assistant](https://www.microsoft.com/en-us/software-download/windows11).
```

Comment:
or the  [Windows 11 Update Assistant](https://www.microsoft.com/en-us/software-download/windows11).

We already say update your system above.